$OpenBSD: patch-src_cfgparse_l,v 1.3 2012/03/23 08:43:54 dcoppa Exp $

Fix cfgparse: leave EAT_WHITESPACE and BINDSYM_COND states on EOL
(upstream git commit 1f6fbaf240351a3559f25bc47286c47b29c1047f)

--- src/cfgparse.l.orig	Fri Mar 23 09:16:58 2012
+++ src/cfgparse.l	Fri Mar 23 09:18:05 2012
@@ -130,6 +130,8 @@ EOL     (\r?\n)
                                   return '[';
                                 }
 <EAT_WHITESPACE>[ \t]*          { yy_pop_state(); }
+<EAT_WHITESPACE>{EOL}           { yy_pop_state(); }
+<BINDSYM_COND>{EOL}             { yy_pop_state(); }
 <WANT_QSTRING>\"[^\"]+\"        {
                                   yy_pop_state();
                                   /* strip quotes */
